courses/fundamentals_of_ada/adv_030_basic_types.rst

```diff
@@ -635,3 +785,37 @@ Ada.Characters.Handling Sample Content
    ...
    end Ada.Characters.Handling;
 
+------
+Quiz
+------
+
+.. code:: Ada
+
+    type T1 is (NUL = 0, A, B, 'C');
+    type T2 is array (Positive range <>) of T1;
+    Obj : T2 := "CC" & A & NUL;
+
+Which of the following propositions are true?
```

Comment:
In the phrasing of this and the previous, the student can't tell if there should be only 1 answer, or 1 or more answers